wallet_transactions/transactions/views.py

```diff
@@ -47,3 +51,33 @@ def transaction_list_for_name(request, name_of_wallet: str):
     if not transaction:
         return Response({"error": "transaction does not exist"})
     return Response(serializer.data)
+
+
+@api_view(["POST"])
+def transaction_create(request):
+    """Create view for transaction which create transaction and chenge the balance in wallets"""
+    user = request.user
+    serializer = TransactionSerializer(data=request.data)
+    serializer.is_valid()
+    if not serializer.is_valid():
+        return Response({"error": "faild"})
+
+    wallet_sender = Wallet.objects.filter(owner__username=user).get(
+        name=serializer.validated_data.get("sender")
+    )
+    wallet_recever = Wallet.objects.get(name=serializer.validated_data.get("receiver"))
+
+    com, transfer, sender_balanse = count_commission(
+        wallet_sender, wallet_recever, serializer
+    )
+
+    if wallet_sender.currency != wallet_recever.currency:
+        serializer.save(status=FAILED, commission=com)
+        return Response({"ERROR": "Transacrion faild"})
```

Comment:
create a separate function for this

Reply:
I realize this function in transaction_send branch